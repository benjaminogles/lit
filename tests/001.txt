
lit author author
lit title title
lit description long description
