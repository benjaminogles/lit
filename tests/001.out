<!doctype html>

<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
<main>
<article>
</article>
</main>
</body>
</html>
